src/components/pages/Bookings/__specs__/BookingsRecap.spec.jsx

```diff
@@ -117,6 +126,126 @@ describe('components | BookingsRecap | Pro user', () => {
     })
   })
 
+  it('should warn user that his prefilters returned no booking when no bookings where returned by selected pre-filters', async () => {
+    // Given
+    loadFilteredBookingsRecap.mockResolvedValue({
+      page: 1,
+      pages: 0,
+      total: 0,
+      bookings_recap: [],
+    })
+    await renderBookingsRecap(props, store)
+
+    // When
+    userEvent.click(screen.getByText('Afficher', { selector: 'button' }))
```

Comment:
```suggestion
    userEvent.click(screen.getByRole('button', { name: 'Afficher' }))
```
Je favoriserais quand même le sélecteur conseillé par testing-library